src/SAML2/SOAPClient.php

```diff
@@ -34,11 +64,11 @@ public function send(Message $msg, SimpleSAML_Configuration $srcMetadata, Simple
         $ctxOpts = array(
             'ssl' => array(
                 'capture_peer_cert' => true,
-                'allow_self_signed' => true
+                'allow_self_signed' => true,
```

Comment:
@sitya Is there a reason this is set beyond testing/debugging purposes? It seems like this should be set to `false` at least by default.